astrowidgets/core.py

```diff
@@ -384,28 +403,50 @@ def is_marking(self):
         """
         return self._is_marking
 
-    @is_marking.setter
-    def is_marking(self, val):
-        if not isinstance(val, bool):
-            raise ValueError('Must be True or False')
-        elif self.click_center and val:
-            raise ValueError('Cannot set to True while in click-center mode')
-        self._is_marking = val
-
-    def stop_marking(self, clear_markers=True):
+    # @is_marking.setter
```

Comment:
I'd rather this is deleted then just commented. We can always check `git history` if we need to see it again, so there is no need to carry this around?

Reply:
oops, forgot about that, fixing